### DISCUSSION TITLE
2024-04-16 CUE Community Update

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#3019] [Community Calls]
We are excited to announce our next CUE Community Update for **Tuesday, April 16, 2024 at 1500 UTC.**

Please reply with any questions/topics you would like to see covered.

### Agenda
- Project updates
- Evaluator updates including performance
- CUE with other languages
- Modules updates
- New cuelang.org and Docs and Content
- cuepls / LSP updates
- Community updates

### Meeting details
https://meet.google.com/nqd-csyi-fmc

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/vDwZoEqEs64

### Slides
https://docs.google.com/presentation/d/1IF3dJzQuwf9aQ9HL_MU_ZtNuH93sCyt8dlT50Y78Z60/edit?usp=sharing

### Auto-generated transcript
https://docs.google.com/document/d/1DWhYLG98MBQVqf0FvuC0FzJLLE0DRQATWQdVJ7d_vAI/edit?usp=sharing

**Comment by dominikdm** [comment]
Added recording, slides and transcript.

**Comment by mhemken-vts** [comment]
Thanks for the update! I wasn't able to join the call, but I went through the slides. Very helpful!

Looking forward to everything in there including the upcoming LSP work!
